src/NLog.Redis/NLog.Redis.csproj

```diff
@@ -14,17 +14,24 @@
     <PackageLicenseUrl>https://raw.githubusercontent.com/richclement/NLog.Redis/master/LICENSE</PackageLicenseUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>git://github.com/richclement/NLog.Redis</RepositoryUrl>
-    
+
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">

```

Comment:
any reason for doing this only on release? IMO always is better (more transparent)

Reply:
Because coverage tools like coverlet break with signing enabled. Other testing tools like NCrunch actually have to disable signing automatically to prevent breaking the signature. It would be nice to use coverage tools like coverlet at some point.

I could disable it only during testing, but that would require introducing a compiler directive - `DEBUG` already exists.

Reply:
Is coverlet better than opencover? I didn't have any issues with the latter and strong naming. 

If we think it's still and issue, I think it would be nice to make it more explicit, e.g `<PropertyGroup Condition=" '$(Configuration)' == 'Coverage' ">`

Reply:
coverlet supports .Netcore, opencover is Windows only. I bet opencover uses the profiler api which doesn't exist under things like mono.